tests/component/db/vertica/test_records_export_options.py

```diff
@@ -0,0 +1,157 @@
+                    self.fail()
+
+    def test_vertica_export_options_datetimeformattz(self):
+        # Vertica doesn't currently allow any configurability on
+        # output datetimeformattz.  Check again before adding any test
+        # cases here!
+        should_raise = {
+            'YYYY-MM-DD HH:MI:SS': True,
+            'YYYY-MM-DD HH24:MI:SSOF': False,
+            'MM/DD/YY HH24:MI': True,
+        }
+        for datetimeformattz in DATETIMEFORMATTZ_CASES:
+            records_format = DelimitedRecordsFormat(variant='vertica',
+                                                    hints={
+                                                        'datetimeformattz': datetimeformattz
+                                                    })
+            unhandled_hints = set(records_format.hints)
+            processing_instructions = ProcessingInstructions(max_failure_rows=123)
+            load_plan = RecordsLoadPlan(processing_instructions=processing_instructions,
+                                        records_format=records_format)
+            # Records Mover passes no particular option for dateformat on
+            # export in Vertica; it always uses YYYY-MM-DD as a result.
+            try:
+                vertica_export_options(unhandled_hints, load_plan)
+            except NotImplementedError:
+                if should_raise[datetimeformattz]:
+                    pass
+                else:
+                    self.fail()
+
+    def test_vertica_export_options_datetimeformat(self):
+        # Vertica doesn't currently allow any configurability on
+        # output datetimeformat.  Check again before adding any test
+        # cases here!
+        should_raise = {
+            'YYYY-MM-DD HH:MI:SS': True,
+            'YYYY-MM-DD HH24:MI:SS': False,
+            'MM/DD/YY HH24:MI': True,
+            'YYYY-MM-DD HH12:MI AM': True,
+        }
+        for datetimeformat in DATETIMEFORMAT_CASES:
+            records_format = DelimitedRecordsFormat(variant='vertica',
+                                                    hints={
+                                                        'datetimeformat': datetimeformat,
+                                                    })
+            unhandled_hints = set(records_format.hints)
+            processing_instructions = ProcessingInstructions(max_failure_rows=123)
+            load_plan = RecordsLoadPlan(processing_instructions=processing_instructions,
+                                        records_format=records_format)
+            # Records Mover passes no particular option for dateformat on
+            # export in Vertica; it always uses YYYY-MM-DD as a result.
+            try:
+                vertica_export_options(unhandled_hints, load_plan)
+            except NotImplementedError:
+                if should_raise[datetimeformat]:
+                    pass
+                else:
+                    raise
+
+    def test_vertica_export_options_datetimeformattz(self):
```

Comment:
F811 redefinition of unused 'test_vertica_export_options_datetimeformattz' from line 45